cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -16,62 +16,122 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == New and changed features
 
-=== Supported binding methods
-* Open vSwitch (OVS) is no longer recommended and should not be used
-in {CNVProductName} 2.0.
-* For the default Pod network, `masquerade` is the only recommended binding
-method. It is not supported for non-default networks.
-* For secondary networks, use the `bridge` binding method.
-
 === Web console improvements
-* You can now view all services associated with a virtual machine in the
-*Virtual Machine Details* screen.
-
 
-== Resolved issues
+* The {product-title} dashboard captures high-level information about clusters.
+From the OpenShift web console, access the dashboard by clicking
+*Home > Dashboards > Overview*. Note that virtual machines are no longer listed
+in the web console project overview. Virtual machines are now listed within the
+*Cluster Inventory* card of the dashboard.
 
-* Deleting a PVC after a CDI import fails no longer results in the importer Pod
-getting stuck in a `CrashLoopBackOff` state. The PVC is now deleted normally.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1673683[*BZ#1673683*])
+=== Other improvements
 
+* After you install {CNVProductName}, MAC pool manager automatically starts.
+If you define a secondary NIC without specifying the MAC address, the MAC pool
+manager allocates a unique MAC address to the NIC.
++
+[NOTE]
+====
+If you define a secondary NIC with a specific MAC address, there is a chance
```

Comment:
s/there is a chance/it is possible 
or something similar, "chance" sounds odd for computing :)